.circleci/config.yml

```diff
@@ -5,172 +5,13 @@ executors:
       - image: docker.mirror.hashicorp.services/cimg/go:1.17
 
 jobs:
-  go-fmt-and-vet:
+  disabled_job:
```

Comment:
We don't need this. I was being too cautious about not disabling CircleCI for all branches before merging this PR. So I had this stub workflow to avoid a failing check from CircleCI on this branch.

I'll remove this and disable the CircleCI webhooks prior to merging this.